docs/source/decoders.md

```diff
@@ -12,7 +12,7 @@ The {class}`decoder.decoders.Decoder` class can also be used with a custom decod
 
 ## Configuring and running the included decoder
 
-To configure the decoder, change the file `settings_decoder.yaml`, which is located by default in the `$HOME/.nds/` folder. You can point the script to use a different configuration file by passing the `--settings-path` argument.
+To configure the decoder, change the file `settings_decoder.yaml`, which is located by default in the `$HOME/.nds/` folder. You can point the script to use a different configuration file by passing the `--config` argument.
```

Comment:
is this parameter `--config` correct? Shouldn't it be `--config-path`?

Reply:
Thanks for the catch! Updated to `--config-dir` and `--config-name`. Strangely `--config-path` is just an alias for `--config-dir`.